### DISCUSSION TITLE
Custom Styling for Credit Card Input not working

### DISCUSSION THREAD

**Discussion by Phainix** [weareseeed/react-square-web-payments-sdk#28] [Help] [answered]
Adding custom css styling to the credit card input doesnt work, it seems the style prop is not recognised.

```
const cardClassSelectors = {
  input: {
    backgroundColor: '#2D2D2D',
    color: '#FFFFFF',
    fontFamily: 'helvetica neue, sans-serif',
  },
};

<CreditCardInput
 style={cardClassSelectors}
 submitButtonId="nextStep"
 overrideStyles={overrideStyles}
/>
```

How can I get this to work, I need the input fields style to match my design theme?

**Comment by danestves** [comment, marked as answer]
Hi @Phainix this is an issue that we have already seen, and we are going to launch the new version in this PR https://github.com/weareseeed/react-square-web-payments-sdk/pull/22 hopefully this week will be the launch!

**Reply by Phainix** [reply]
This is great to note @danestves, will be looking out for the launch!

**Reply by danestves** [reply]
I will pin you as soon as I get the chance that we have the `v2.3.0` on NPM

**Reply by JeremiahPinto** [reply]
Hey guys, any updates on this issue? Am in the same boat.

**Comment by danestves** [comment]
A new release has been published with the feat of change the style of the card [`v2.3.0`](https://github.com/weareseeed/react-square-web-payments-sdk/releases/tag/v2.3.0)
